packages/modelfusion/tsup.config.ts

```diff
@@ -4,21 +4,18 @@ export default defineConfig([
   {
     entry: ["src/index.ts"],
     format: ["cjs", "esm"],
-    dts: true,
```

Comment:
Does the export still produce the typescript dts files when this is removed?

Reply:
Yes it does.  It was necessary to remove this so that dts could be disabled when the .d.ts.map files get produced via `pnpm build:dtsMap` and `pnpm dev`.  I see it made sense in context of that other PR that you commented on: https://github.com/lgrammel/modelfusion/pull/282#issuecomment-1952979676.